examples/08_experimental/plot_surface_image_and_maskers.py

```diff
@@ -5,11 +5,13 @@
 copied from the nilearn sandbox discussion, to be transformed into tests &
 examples
 
-NOTE this example is meant to support discussion around a tentative API for
-surface images in nilearn. This functionality is provided by the
-nilearn.experimental.surface module; it is still incomplete and subject to
-change without a deprecation cycle. Please participate in the discussion on
-GitHub!
+.. note:
```

Comment:
```suggestion
.. note::
```
Otherwise it won't render as in: https://output.circle-artifacts.com/output/job/a6a35138-d22d-41c6-a330-55c36f3b9771/artifacts/0/dev/auto_examples/08_experimental/plot_surface_image_and_maskers.html#sphx-glr-auto-examples-08-experimental-plot-surface-image-and-maskers-py